### PR TITLE
Sarek wes defaults upps

### PR DESCRIPTION
**Pull request by monikaBrandt**
In this PR the following changes are included:
- When running Sarek the parameter --generate_gvcf will be added as default.
- A new README.md for WES analysed with Sarek is now available.
- A new TACA-config for staging of WES analysed with Sarek is now available.

### CODE REVIEW

**Comment by monikaBrandt** [comment]
Apparently I'm not allowed to set Reviewers on this PR. It would be great if someone from Uppsala and someone from Stockholm can have a look at it. Maybe @aanil and @b97pla can help me with code review? :)

File: roles/taca/tasks/main.yml [hunk 1]
```diff
@@ -69,6 +72,9 @@
 - name: Deploy sthlm sarek config
   template: src="site_taca_sarek_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_sarek_delivery.yml"
 
+- name: Deploy sthlm sarek WES config
+  template: src="site_taca_sarek_wes_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_sarek_wes_delivery.yml"
```
Comment (b97pla) [inline roles/taca/tasks/main.yml:76]: Not sure if Sthlm will use this, so perhaps this task is not needed?

File: roles/taca/templates/site_taca_sarek_wes_delivery.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            -
+              required: True
+        -
+            - {{ ngi_resources }}/TACA/apply_recalibration.sh
+            - <STAGINGPATH>/Resources/
+            -
+              required: True
+              no_digest_cache: True
+        -
+            - {{ ngi_resources }}/TACA/{{ site }}/DELIVERY.README.SAREK.WES.j2
```
Comment (b97pla) [inline roles/taca/templates/site_taca_sarek_wes_delivery.yml.j2:69]:
```suggestion
            - {{ ngi_resources }}/TACA/{{ site }}/DELIVERY.README.SAREK.WES.md
```

File: roles/taca/templates/site_taca_sarek_wes_delivery.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            - <STAGINGPATH>/
+            -
+              required: True
+              no_digest_cache: true
```
Comment (b97pla) [inline roles/taca/templates/site_taca_sarek_wes_delivery.yml.j2:73]:
```suggestion
              no_digest_cache: True
```

**Review by b97pla — CHANGES_REQUESTED** [review]
Looks good so far but needs a few fixes. Primarily, you will need to add a task to `roles/sarek/tasks/main.yml` that will deploy the README to the TACA folder. See: 
https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/32429494846039d2b3db1118c9787b2a9a2e3e70/roles/sarek/tasks/main.yml#L56

File: roles/sarek/templates/DELIVERY.README.SAREK.WES.md.j2 [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# DELIVERY OF RESULTS FROM EXOME ANALYSIS WITH SAREK
```
Comment (aanil) [inline roles/sarek/templates/DELIVERY.README.SAREK.WES.md.j2:1]: This file needs to be generated in the tasks.yml file in the sarek role, like here
https://github.com/NationalGenomicsInfrastructure/irma-provision/blob/234a129597be23bb8f37064be81f9a87b616d53a/roles/sarek/tasks/main.yml#L56

File: roles/taca/tasks/main.yml (hunk 1 above)
Comment (aanil) [reply roles/taca/tasks/main.yml:76]: I think we'll use this, but it may require some modification. We can address that in another PR maybe.

**Comment by monikaBrandt** [comment]
Thank you @aanil and @b97pla for your valuable comments! There is a new commit available to this PR that hopefully will fix the small errors.

**Review by b97pla — APPROVED** [review]
looks good :+1:

**Pull request merged**